drivers/gles3/rasterizer_storage_gles3.cpp

```diff
@@ -8395,6 +8439,23 @@ void RasterizerStorageGLES3::initialize() {
 
 	config.use_physical_light_attenuation = GLOBAL_GET("rendering/quality/shading/use_physical_light_attenuation");
 
+	config.use_occlusion_queries = bool(GLOBAL_GET("rendering/quality/occlusion_queries/enable"));
+	// I <3 different gpu drivers! *screams in agony*
```

Comment:
lol, funny, but should be removed before merging

Reply:
That shouldn't be there. Sorry. At least I managed to find & remove all the more explicit notes I left while working.